contracts/DebtLocker.sol

```diff
@@ -69,27 +78,27 @@ contract DebtLocker is IDebtLocker, DebtLockerStorage, MapleProxied {
         return _repossessed ? _handleClaimOfRepossessed() : _handleClaim();
     }
 
-    function setAllowedSlippage(uint256 allowedSlippage_) external override {
+    function setAllowedSlippage(uint256 allowedSlippage_) external override live {
         require(msg.sender == _getPoolDelegate(), "DL:SAS:NOT_PD");
 
         emit AllowedSlippageSet(_allowedSlippage = allowedSlippage_);
     }
 
-    function setAuctioneer(address auctioneer_) external override {
+    function setAuctioneer(address auctioneer_) external override live {
         require(msg.sender == _getPoolDelegate(), "DL:SA:NOT_PD");
 
         Liquidator(_liquidator).setAuctioneer(auctioneer_);
 
         emit AuctioneerSet(auctioneer_);
     }
 
-    function setMinRatio(uint256 minRatio_) external override {
+    function setMinRatio(uint256 minRatio_) external override live {
```

Comment:
Setters should be allowed during paused state, since:
- there is to harm setting values during a paused state (and so no need to also waste gas during live states)
- the fix might require a setter, during a paused state